packages/gui/components/SwipeDownGesture.tsx

```diff
@@ -17,7 +21,7 @@ import { AnimatedView } from './aliased';
 
 interface Props {
 	children: ReactNode;
-	style?: ViewStyle;
+	style?: ViewStyle | ViewStyle[];
```

Comment:
In the use of this style, it's already used in an array with animatedStyle, you need to check if it's possible to use array of styles for this prop